cql/DashboardLibrary.cql

```diff
@@ -982,6 +986,51 @@ define PertinentImmunizationsSummary:
     edited: HasBeenLocallyEdited(I)
   }
 
+//------------------------------------------------------------------------------
+// PERTINENT EPISODES OF CARE
+//------------------------------------------------------------------------------
+// Episodes of care related to cervical cancer
+//    Allow all clinical statues and no lookback restriction // TODO: Modify this
+
+define function HasBeenLocallyEdited(E EpisodeOfCare):
+  if IsNull(E.identifier) then false
+  else AnyTrue( (E.identifier) I return I.system = 'http://OUR-PLACEHOLDER-URL.com' )
+
+define PregnancyEpisodesOfCare:
```

Comment:
Use SNOMED CT code

Reply:
SNOMED-CT code was added in. I restructured this Retrieve to use the new SNOMED-CT code.